test/playbooks_reader_test.rb

```diff
@@ -5,89 +5,63 @@
+      end
 
-  test 'should return playbooks names as an array' do
-    playbooks_names = %w[ibm.spectrum_virtualize.generic_info.yml ibm.spectrum_virtualize.generic_ansible_sample.yaml xprazak2.forklift_collection.katello_devel.yml xprazak2.forklift_collection.keycloak.yml xprazak2.forklift_collection.rackspace_hostname.yml xprazak2.forklift_collection.etc_hosts_localhost.yml xprazak2.forklift_collection.luna-devel.yml xprazak2.forklift_collection.katello_client.yml xprazak2.forklift_collection.leapp_devel.yml xprazak2.forklift_collection.custom_certificates.yml xprazak2.forklift_collection.libvirt.yml xprazak2.forklift_collection.bats.yml xprazak2.forklift_collection.collect_debug.yml xprazak2.forklift_collection.smoker.yml xprazak2.forklift_collection.foreman_proxy_content_dev.yml xprazak2.forklift_collection.katello.yml xprazak2.forklift_collection.luna_demo_environment.yml xprazak2.forklift_collection.robottelo.yml xprazak2.forklift_collection.foreman_provisioning.yml xprazak2.forklift_collection.rpm_packaging.yml xprazak2.forklift_collection.upgrade.yml xprazak2.forklift_collection.dynflow_devel.yml xprazak2.forklift_collection.foreman_proxy_content.yml xprazak2.forklift_collection.freeipa_server.yml xprazak2.forklift_collection.setup_forklift.yml xprazak2.forklift_collection.luna.yml xprazak2.forklift_collection.fips.yml xprazak2.forklift_collection.squid.yml xprazak2.forklift_collection.foreman.yml xprazak2.forklift_collection.setup_user_devel_environment.yml xprazak2.forklift_collection.foreman_platform.yml xprazak2.forklift_collection.hammer_devel.yml xprazak2.forklift_collection.kubevirt.yml xprazak2.forklift_collection.luna_provisioning.yml xprazak2.forklift_collection.collect_debug_draft.yml xprazak2.forklift_collection.new.yml xprazak2.forklift_collection.new2.yml]
-    Dir.expects(:glob).with('/usr/share/ansible/collections/ansible_collections/*/*/playbooks/*').returns([])
-    Dir.expects(:glob).with('/etc/ansible/collections/ansible_collections/*/*/playbooks/*').returns(playbooks_names)
-    res = Proxy::Ansible::PlaybooksReader.playbooks_names
-    assert_equal Array, res.class
-    assert_equal 37, res.count
+      it 'should return playbooks array of all existing playbooks' do
+        File.expects(:read).with(listings[2]).returns(playbook1)
+        res = Proxy::Ansible::PlaybooksReader.playbooks(nil)
+        assert_equal Array, res.class
+        assert_equal 3, res.count
+      end
+    end
   end
 
-  test 'raises error if the playbooks path is not readable' do
-    Proxy::Ansible::PlaybooksReader.expects(:read_collection_playbooks)
-                                   .with('/etc/ansible/collections', nil).raises(Proxy::Ansible::ReadPlaybooksException.new('Could not read Ansible roles'))
-    ex = assert_raises(Proxy::Ansible::ReadPlaybooksException) do
-      Proxy::Ansible::PlaybooksReader.playbooks(nil)
+  describe 'playbooks_names method' do
+    let(:ansible_config) { fixtures['ansible_config'] }
```

Comment:
Should we remove these now? it is the same as the global one, so no need to override it now, right?